### ISSUE TITLE
Custom scripts in head section

### ISSUE THREAD

**Issue by mikasjp** [thelehhman/plainwhite-jekyll#45]
Users should be able to include their own JS and CSS scripts in the head section.

**Comment by thelehhman** [comment]
I don't see the point of this. If you want to include your own JS, CSS just add them to your fork. What do you suggest?

**Comment by mikasjp** [comment]
Imagine this situation. For example, the user wants to install the `Hotjar` service on their website. To achieve this, he must embed an external `JS` script in the `head` section. This is currently not possible. Some services also require `CSS` scripts.

**Comment by DotIN13** [comment]
> Imagine this situation. For example, the user wants to install the `Hotjar` service on their website. To achieve this, he must embed an external `JS` script in the `head` section. This is currently not possible. Some services also require `CSS` scripts.

You can always fork this repo and make changes to `head.html` partial anyway you like, then push to your own repo.

**Comment by thelehhman** [comment]
Yes, I agree with @DotIN13. See [this](https://github.com/thelehhman/thelehhman.github.io/blob/master/_includes/head.html) as an example.
